docs/software-platforms/aws/working-with-s3-buckets.md

```diff
@@ -40,4 +40,69 @@ You can then use `$OPENSCPCA_RESULTS_BUCKET` (or, `${OPENSCPCA_RESULTS_BUCKET}`)
 
 ## Syncing your results to S3
 
-_Coming up in the next PR._
+We have written a script to help you sync your results to your S3 bucket, stored in [`scripts/sync-results.py`](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/scripts/sync-results.py).
+For this script to work, you need to be [logged into the AWS account profile you use for contributing to OpenScPCA](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
+
+This script syncs the contents of a given analysis module's `results` and `plots` directories from your computer to S3.
+It does not also sync S3 contents back to your computer.
+Your files will be synced to a bucket called `s3://{your bucket name}/{analysis module name}`, where:
+
+- Results will be synced to `s3://{your bucket name}/{analysis module name}/results`
+- Plots will be synced to `s3://{your bucket name}/{analysis module name}/plots`
+
+
+The simplest usage of this script, called from the `OpenScPCA-analysis` repository root folder, is:
+
+```sh
+scripts/sync-results.py \
+    --module {analysis module name} \
+    --bucket {name of your bucket}
+```
+
+- `--module` (or `-m`) is the folder name of the analysis module whose results you want to sync
+- `--bucket` (or `-b`) is your [bucket name](#finding-your-bucket-name)
+  - You can omit this argument if you have saved your bucket name in the environment variable `OPENSCPCA_RESULTS_BUCKET`
+
+**Before you run the script to sync your results,** we recommend you first run the script with the `--dryrun` flag.
+This flag will not actually sync your files, but it will instead print out which files will be synced, and where, to your S3 bucket:
+
+```sh
+scripts/sync-results.py \
+    --module {analysis module name} \
+    --bucket {name of your researcher bucket} \
+    --dryrun
+```
+
+If everything looks correct to you when running with `--dryrun`, you can then proceed to run the script without this flag and actually perform the syncing.
+
+### Additional script flags
+
+By default, there any result or plot files that exist on S3 but that you have locally deleted, this script will _not also delete_ those files from S3.
+To override this behavior and delete them from S3 as well, use the `--destructive-sync` flag:
```

Comment:
Worth saying that this will not delete anything from other modules, I think
```suggestion
To override this behavior and delete these files from S3 as well (but only for the specified analysis module), use the `--destructive-sync` flag:
```